spec/unit/cli_spec.rb

```diff
@@ -183,17 +190,12 @@
         run_command(["inspect", "--exclude-scope=packages,repositories", example_host])
       end
 
-      it "forwards the --skip-files option to the InspectTask" do
-        expect_any_instance_of(InspectTask).to receive(:inspect_system).
-          with(
-            an_instance_of(SystemDescriptionStore),
-            example_host,
-            example_host,
-            an_instance_of(CurrentUser),
-            Inspector.all_scopes,
-            skip_files: "/foo/bar"
-          ).
-          and_return(description)
+      it "forwards the --skip-files option to the InspectTask as an unmanaged_files filter" do
+        expect_any_instance_of(InspectTask).to receive(:inspect_system) do |_instance, _store,
```

Comment:
Block argument expression is not on the same line as the block start.<br>Extra space before block parameter detected.